content-services/community/develop/sdk.md

```diff
@@ -746,22 +746,22 @@ The following table explains some of these properties:
 | acs.debug.port | `number` | 8888 | The external port (i.e. outside container) for ACS Repository remote debugging.|
 | postgres.port | `number` | 5555 | The external port (i.e. outside container) for PostgreSQL database.|
 
-There are some Alfresco Share related properties listed here, but they are not used unless you uncomment some code in the 
+There are some Alfresco Share related properties listed here, but they are not used unless you uncomment some code in the
```

Comment:
Is this the terminology? 

```suggestion
There are some Alfresco Share related properties listed here, but they are not used unless you comment out some of the code in the
```
```suggestion
There are some Alfresco Share related properties listed here, but they are not used unless you uncomment some code in the
```

Reply:
It should be uncomment actually, not comment out